src/seedu/addressbook/data/person/Address.java

```diff
@@ -52,6 +52,13 @@ public int hashCode() {
         return value.hashCode();
     }
 
+    @Override
+    public String getPrintableString() {
+        String prefix = "Address: ";
+        String printAddress = prefix.concat(value);
+        return printAddress;
```

Comment:
Can be written in one line: `return prefix + value`. Also the prefix can be declared as a constant.